get-started/guide-neo4j-browser/guide-neo4j-browser.adoc

```diff
@@ -97,174 +147,111 @@ m|:help keys
 
-You can click the small query above the graph visualisation to pull it back into the editor.
+kbd:[Ctrl+Up] and kbd:[Ctrl+Down] (Mac users, use the kbd:[Cmd] key) allows you to navigate command history, and you can access all command history with `:history`.
+The command history is persisted across browser restarts.
 
-Use the keyboard shortcuts listed above to work efficiently with the editor area.
+You can switch between `Graph`, `Table`, `Text`, and `Code` views to see the results in various formats by clicking the icons on the left of each pane.
 
-Navigate input history with kbd:[Ctrl+Up] and kbd:[Ctrl+Down], access all of it via `:history`. 
-The history will be persisted across browser restarts.
+****
+[NOTE] 
+Don't worry if you don't see any output.
+You might just be in Graph mode, but had your query return tabular/scalar data.
+To see the results, just switch the mode to the Table view.
+****
 
-You can switch between tabular, visual mode, query plan and x-ray mode for results with the icons on the left of each panel, 
+Query time is reported in the `Table` or `Code` views (*don't rely on that exact timing though*), and it includes the latency and deserialization costs, not just the actual query execution time.
 
-[NOTE] 
-Don't worry if you don't see any output, you might just be in visual mode but returned tabular/scalar data, just switch the mode to tabular
+You can also download the results as a CSV from the `Table`, `Text`, and `Code` views or download the graph as an SVG or PNG from the `Graph` view, as shown below.
 
+.Download graph
+image:{img}download_graph.png[]
 
-Query time is reported in the tabular view, *don't rely on that exact timing though* it includes the latency and (de-)serialization costs, not just the actual query execution time.
+.Download CSV
+image:{img}download_csv.png[]
 
-You can download the results as CSV from the tabular output panel (top right download icon), and as JSON (download icon above the panel).
-The graph visualization can be exported as PNG and SVG.
+If you enter fullscreen mode of a graph visualization, you can zoom in and out.
+After a node is clicked, it gets a halo, where you can expand and remove nodes from the visualization.
+You can also turn previously dragged nodes loose again.
 
-If you enter fullscreen mode of a graph visualization you can zoom in and out.
-After a node is clicked it gets a halo, in which you can expand and remove nodes from the visualization. 
-You can also turn previously dragged nodes lose again.
+== Graph Metadata
 
-== Meta Graph
+In the left-side menu, the top icon is the database section (icon:database[]), where you can find the *currently used* node labels, relationship types, and property keys.
+Clicking on any one of those options runs a quick query to show you a sample of the graph with those elements.
 
```

Comment:
mention the additional canned/frequently used queries that also include a visual "schema" query. (i.e. `call db.schema()`

Reply:
Is there a list of those somewhere? How do they know what to run?